modules/gdscript/gdscript_parser.cpp

```diff
@@ -2774,6 +2774,24 @@ GDScriptParser::ExpressionNode *GDScriptParser::parse_get_node(ExpressionNode *p
 			get_node->chain.push_back(identifier);
 		} while (match(GDScriptTokenizer::Token::SLASH));
 		return get_node;
+	} else if (match(GDScriptTokenizer::Token::SLASH)) {
+		advance();
```

Comment:
I think I wasn't clear about this, sorry, but the `advance()` also needs to be removed, since `match()` already does that.